code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -1,39 +1,51 @@
-/mob/living/carbon/human/getarmor(def_zone, type)
+/mob/living/carbon/human/getarmor(def_zone, type, penetration)
 	var/armorval = 0
 	var/organnum = 0
 
 	if(def_zone)
 		if(isbodypart(def_zone))
 			var/obj/item/bodypart/bp = def_zone
 			if(bp)
-				return checkarmor(def_zone, type)
+				return checkarmor(def_zone, type, penetration)
 		var/obj/item/bodypart/affecting = get_bodypart(check_zone(def_zone))
 		if(affecting)
-			return checkarmor(affecting, type)
+			return checkarmor(affecting, type, penetration)
 		//If a specific bodypart is targetted, check how that bodypart is protected and return the value.
 
 	//If you don't specify a bodypart, it checks ALL your bodyparts for protection, and averages out the values
 	for(var/obj/item/bodypart/BP as() in bodyparts)
-		armorval += checkarmor(BP, type)
+		armorval += checkarmor(BP, type, penetration)
 		organnum++
 	return (armorval/max(organnum, 1))
 
-
-/mob/living/carbon/human/proc/checkarmor(obj/item/bodypart/def_zone, d_type)
+/// Check the armour for a specified bodyzone and damagetype.
+/// Returns a value with 0 representing 0% protection and 100 representing 100% protection.
+/// 50 + 50 = (1 - (1 * 0.5 * 0.5) = 75%)
+/// 100 + 50 = (1 - (1 * 0 * 0.5) = 100%)
+/// 50 + (-50) = (1 - (1 * 0.5 * 1.5) = 25%)
+/// 100 + (-50) = (1 - (1 * 0 * 1.5) = 100%)
+/// -50 = (1 - (1 * 1.5) = -50%)
+/// Any armour that exceeds 100% protection will be clamped down to 100%
```

Comment:
Is it possible to go above 100%?
Does any role have available armor that would exceed 100%? (mostly thinking about nukies and their armors)

Reply:
Only the doomguy armour gets over 100% armour. Nukie goes up to like 60%.